common/src/main/java/com/revenuecat/purchases/common/networking/ExtraHeadersManager.kt

```diff
@@ -0,0 +1,16 @@
+package com.revenuecat.purchases.common.networking
+
+class ExtraHeadersManager {
+    private val _extraHeaders = mutableMapOf<String, String>()
+
+    val extraHeaders: Map<String, String>
+        get() = _extraHeaders
+
+    fun setBackwardsCompatibleOverride(enabled: Boolean) {
+        if (enabled) {
+            _extraHeaders["X-RC-BW-Only-Override"] = "true"
```

Comment:
nitpick: I'd maybe name this something that looks more obvious at a glance for someone with no context, like `X-Force-BillingClient4-APIs` or something. That being said, it's not even technically BC4 when it's backend-side, so... feel free to disregard. Naming this is hard.

Reply:
This is less about Billing Client version and more about not supporting "Product Details" so maybe something like `X-RC-Product-Details-Not-Supported`??? Which is essentially what backwards compatible I guess 🤷‍♂️ 

But I'm not super tied to names users/developers aren't really seeing it 😛